pkg/nimble/autoconn/nimble_autoconn.c

```diff
@@ -297,8 +310,8 @@ int nimble_autoconn_update(const nimble_autoconn_params_t *params,
 
 void nimble_autoconn_enable(void)
 {
-    DEBUG("[autoconn] ACTIVE\n");
     if (nimble_netif_conn_count(NIMBLE_NETIF_UNUSED) > 0) {
+        DEBUG("[autoconn] ACTIVE\n");
```

Comment:
Unrelated?

Reply:
its an improvement, it was actually kind of broken before...